projects/jobs/jobs.groovy

```diff
@@ -1,7 +1,3 @@
-// Constants
-def gerritBaseUrl = "ssh://jenkins@gerrit:29418"
-def cartridgeBaseUrl = gerritBaseUrl + "/cartridges"
```

Comment:
Removed cartridgeBaseUrl as it is no longer used since the introduction of PR https://github.com/Accenture/adop-platform-management/pull/32